yarn.lock

```diff
@@ -10695,6 +10686,15 @@ schema-utils@^0.4.4:
     ajv "^6.1.0"
     ajv-keywords "^3.1.0"
 
+schema-utils@^1.0.0:
```

Comment:
It's not in the dependency tree of next after this change, only when using next-css it's there because of used plugins in that module.